validate.lic

```diff
@@ -57,6 +57,14 @@ class DRYamlValidator
     warn('You have no weapons configured in weapon_training: this will likely cause problems.')
   end
 
+  def assert_that_cyclic_no_release_spell_is_invalid_unless_listed_in_base_spells(settings)
+    return if !settings.cyclic_no_release
+
+    settings.cyclic_no_release
+      .reject { |spell_name| get_data('spells').spell_data.include?(spell_name) }
```

Comment:
💡 Avoid reloading the file for each spell being checked. The `reject` method is like a loop, it iterates all elements listed in `cyclic_no_release` list. The result of `get_data('spells')` won't be different each iteration.
```suggestion
    spell_data = get_data('spells').spell_data
    settings.cyclic_no_release
      .reject { |spell_name| spell_data.include?(spell_name) }
```

Reply:
Thank you. I did not know that. Commited.

Reply:
Nice catch!